rc/filetype/php.kak

```diff
@@ -39,7 +39,12 @@ add-highlighter shared/php/doc_comment2  region /\*\*  \*/             ref php/d
 add-highlighter shared/php/comment1      region //     '$'             fill comment
 add-highlighter shared/php/comment2      region /\*    \*/             fill comment
 add-highlighter shared/php/comment3      region '#'    '$'             fill comment
-
+# Example PHP heredoc:
```

Comment:
I do not think this comment is very valuable, this is not really the place to document php's syntax, and the highlighter name (shared/php/heredoc) should be enough to understand what this is supposed to highlight.

Reply:
Done -- removed the comment

Reply:
@mawww ^